src/database/migrations/20220708100354-create-chat.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('chats', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      userName: {
```

Comment:
you should keep track of users not just random username which means user has to be logged in order to to know who is who